src/controllers/react.controller.js

```diff
@@ -1,13 +1,26 @@
 const httpStatus = require('http-status');
 
 const { i18n } = require('../config');
-const { React } = require('../models');
+const { React, Moment } = require('../models');
 const { ApiError, catchAsync } = require('../utils');
 
-const createReaction = catchAsync(async (req, res, next) => {
+const sendReaction = catchAsync(async (req, res, next) => {
   const userId = req.user._id;
-  //#TODO : check post
-  const reaction = await React.create({ userId, ...req.body });
+  const momentId = req.body.momentId;
+  const momentExisting = await Moment.findById(momentId);
+  if (!momentExisting) {
+    throw new ApiError(httpStatus.NOT_FOUND, i18n.translate('moment.momentNotFound'));
+  }
+
+  const reaction = await React.findOne({ userId, momentId });
+
+  if (reaction) {
+    reaction.react.push(req.body.react);
```

Comment:
```suggestion
    reaction.reacts.push(req.body.react);
```
em cập nhật lại model trường này là mảng thêm s vào á